daemon/src/model/cfd.rs

```diff
@@ -241,8 +241,8 @@ pub enum NoRolloverReason {
     Attested,
     #[error("Cannot roll over while CFD is in collaborative settlement")]
     InCollaborativeSettlement,
-    #[error("Cannot roll over when CFD is final")]
-    Final,
```

Comment:
as per your commit message: an alternative way we could deal with appending events after "closing" cfd would be just treating the *first* closure as the one that got in (in other words, it could be handled in during rehydration phase - it doesn't need to look at the last event in all cases)

Reply:
I think this goes against the paradigm of even-sourcing. We would start "ignoring" events. I think it's better to keep the even history correct. We could use it as a workaround to deal with `Cfds` that already are in a "wrong" state, but I think those should fix itself, once we get a confirmation on chain, because the confirmation will actually close it through finality.

Reply:
I'm happy to see work toward that in a follow up PR! I think it's an interesting idea